docs/source/getting_started/installation/ai_accelerator/neuron.inc.md

```diff
@@ -116,12 +116,9 @@ Once neuronx-cc and transformers-neuronx packages are installed, we will be able
 ```console
 git clone https://github.com/vllm-project/vllm.git
 cd vllm
-pip install -U -r requirements-neuron.txt
 VLLM_TARGET_DEVICE="neuron" pip install .
```

Comment:
don't you still need "requirements-neuron.txt"?

Reply:
Well, they are install requirements for the vllm neuron package, so it's an extra redundant step.